src/main/java/com/epam/lowcost/DAO/UserDAOImpl.java

```diff
@@ -0,0 +1,125 @@
+             Statement stm = connection.createStatement();
+             ResultSet rs = stm.executeQuery(sql)) {
+            if (rs.next()) {
+                long id = rs.getLong("id");
+                String email = rs.getString("email");
+                String password = rs.getString("password");
+                boolean isAdmin = rs.getBoolean("isAdmin");
+                String firstName = rs.getString("firstName");
+                String lastName = rs.getString("lastName");
+                String documentInfo = rs.getString("documentInfo");
+                LocalDateTime birthday = rs.getTimestamp("birthday").toLocalDateTime();
+
+                return user = new User(id, email, password, isAdmin, firstName, lastName, documentInfo, birthday);
+            }
+
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return user;
+    }
+
+    @Override
+    public User addUser(User user) {
+        DateTimeFormatter f = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm:ss");
+        String sql = String.format(
+                "INSERT INTO USERS (email, password, isAdmin, " +
+                        "firstName, lastName, documentInfo, birthday) " +
+                        "VALUES ('%s', '%s',%b,'%s','%s','%s','%s')",
+
+                user.getEmail(), user.getPassword(), user.isAdmin(),
+                user.getFirstName(), user.getLastName(), user.getDocumentInfo(),f.format(user.getBirthday()));
+
+        try (Connection connection = dataSource.getConnection();
+             Statement stm = connection.createStatement()
+             ) {
+            int insert = stm.executeUpdate(sql);
+            if (insert == 1){
+                ResultSet rs = stm.executeQuery("SELECT * FROM USERS");
+                rs.last();
+                long newId = rs.getLong("id");
+                user.setId(newId);
+                return user;
+            }
+
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        user = null;
+        return user;
+    }
+
+    @Override
+    public User updateUser(User user) {
+        DateTimeFormatter f = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm:ss");
+        String sql = String.format(
+                "UPDATE USERS SET email='%s',password='%s',isAdmin='%b',firstName='%s'," +
+                "lastName='%s',documentInfo='%s',birthday='%s' WHERE id=%d",
+                user.getEmail(), user.getPassword(), user.isAdmin(),user.getFirstName(),
+                user.getLastName(), user.getDocumentInfo(),f.format(user.getBirthday()),user.getId());
+        try(Connection conn = dataSource.getConnection();
+            Statement stm  = conn.createStatement()){
+                int update = stm.executeUpdate(sql);
+                if(update == 1){
+                    return user;
+                }
+        }catch (SQLException e){
+            e.printStackTrace();
+        }
+        user = null;
```

Comment:
always return null ?

Reply:
Decided to let it be like this for a while.